commands/index.js

```diff
@@ -16,31 +16,57 @@ function generateRandomSessionId() {
   return Math.floor((Math.random() * 9999) + 1).toString();
 }
 
-function runNLPResponse(bot, message) {
-  const request = app.textRequest(message.text, {
+function getCurrencySymbol(currency) {
+  return config.CURRENCY_SYMBOLS[currency] ? config.CURRENCY_SYMBOLS[currency] : '';
```

Comment:
Yep. `undefined` is [falsy.](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)